sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_lease.py

```diff
@@ -265,9 +260,9 @@ def break_lease(self, lease_break_period=None, **kwargs):
         :return: Approximate time remaining in the lease period, in seconds.
         :rtype: int
         """
-        self._blob_lease_client.break_lease(lease_break_period=lease_break_period, **kwargs)
+        return self._blob_lease_client.break_lease(lease_break_period=lease_break_period, **kwargs)
```

Comment:
Hmmm, I think is how the code _should_ have been, but it's crazy to me that this wasn't returning anything before? Despite documentation stating otherwise? Probably fine to take the change but crazy that this wasn't returning at all before?? I would even go as far as say we should add a test for this API since clearly we don't have one atm...

Reply:
Interesting, yeah, this would be a bug and it's good to fix it up. I imagine we don't usually look at the return value for `break_lease` as that's not super relevant. Customer's probably do not either.

Reply:
All great points, and 100% agree. Blob sync + aio `break_lease` APIs return integer value representing the approximate time in the lease period (in seconds). The lease client is making a generated code call to complete the operation, which returns that integer value.

So to make it consistent I added the return statement. 

It's definitely a surprising bug, luckily no tests use this return value but if customers do in the future they'll get the expected value now :)

Reply:
Our other packages also return an integer value for all `break_lease` APIs.